### DISCUSSION TITLE
How can I create a repository and merge it

### DISCUSSION THREAD

**Discussion by vivekvardhan2810** [community/community#84970] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

pls hlp

**Comment by avaniishh123** [comment, marked as answer]
I’m from Repo A and B in the README.md file looks good. I thought of adding repo named mars, and README file would have contained I am from Mars.But this is the exercise which you can do.

Fork repo-c.
Create a new repo named mars or it can be your city or country name.
Merge mars repo to forked repo-c.
Create the pull request to repo c.
You see, the merging of one or multiple repositories into single repositories is simple. In the process of merging repositories, the only thing that’s going t

**Comment by vivekvardhan2810** [comment]
thank you avaniish
